packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -63,6 +77,12 @@ export default class VTable {
                                 width: hasTd ? rect.width : undefined,
                                 height: hasTd ? rect.height : undefined,
                             };
+                            if (td.classList.contains(TABLE_CELL_SELECTED)) {
+                                this.selection.firstCol = this.selection.firstCol ?? targetCol;

```

Comment:
this.selection.firstCol will be treated as false if its value is 0.
Same for this.selection.firstRow

Reply:
Only when firstCol or firstRow are null is going to use the right Expr